reference/6/Microsoft.PowerShell.Core/About/about_Logging.md

```diff
@@ -0,0 +1,144 @@
+
+Unlike Linux or MacOS, Windows requires the event provider to be registered
+before logged events can appear in the event log.  For PowerShell, this is
+accomplished by running the `RegisterManifest.ps1` from an elevated
+PowerShell prompt.
+
+The script is located in the the install directory and should be run from that location.
+
+### Unregistering the PowerShell event provider on Windows
+
+Registering an event provider places a lock in the binary containing the data
+needed to decode events. To update this binary, the provider must first be
+unregistered to release this lock.
+
+To unregister the PowerShell provider, run the `RegisterManifest.ps1` from an
+elevated PowerShell prompt and specify the `-Unregister` switch.  Once the
+upgrade of PowerShell has completed, run `RegisterManifest.ps1` a second time
+to register the updated ETW provider.
+
+## Viewing PowerShell log output on Linux
+
+PowerShell logs the syslog on Linux and any of the tools commonly used to view
+syslog contents may be used.
+
+The format of the log entries uses the following template:
+
+```
+TIMESTAMP MACHINENAME powershell[PID]: (COMMITID:TID:CID) [EVENTID:TASK.OPCODE.LEVEL] MESSAGE
+```
+
+| Field       | Description
+|-------------|---|
+| `TIMESTAMP`   | A date/time when the log entry was produced.
+| `MACHINENAME` | The name of the system where the log was produced.
+| `PID`         | The process id of the process that wrote the log entry.
+| `COMMITID`    | The git commit id or tag that was used to produce the build.
+| `TID`         | The thread id of the thread that wrote the log entry.
+| `CID`         | The hex channel identifier of the log entry.<br/>10 = Operational, 11 = Analytic
+| `EVENTID`     | The event identifier of the log entry.
+| `TASK` | The task identifier for the event entry
+| `OPCODE` | The opcode for the event entry
+| `LEVEL` | The log level for the event entry
+| `MESSAGE` | The message associated with the event entry
+|
+
+* NOTE:EVENTID, TASK, OPCODE, and LEVEL are the same values as used when
+logging to the windows event log.
+
+### Filtering PowerShell log entries using rsyslog
+By default, PowerShell log entries are written to the default location/file
+for syslog.  However, it is possible to redirect the entries to a custom
+file.
+
+* Create a conf for PowerShell log configuration and provide a number that
+is less than 50 (for `50-default.conf`), such as `40-powershell.conf`. The
+file should be placed under `/etc/rsyslog.d`.
+
+* Add the following entry to the file
+```
+:syslogtag, contains, "powershell[" /var/log/powershell.log
+& stop
+```
+* Ensure `/etc/rsyslog.conf` includes the new file. Often, it will have
+a generic include statement that looks like following:
+
+`$IncludeCnofig /etc/rsyslog.d/*.conf`
+
+If it does not, you will need to add an include statement manually.
+
+* Ensure attributes and permissions are set appropriately
+
+```
+-rw-r--r-- 1 root root   67 Nov 28 12:51 40-powershell.conf
```

Comment:
Should show example `chmod` command for appropriate permissions

Reply:
fixed